### ISSUE TITLE
Terraform 0.6.6:  ssh fingerprint does not work on DigitalOcean droplet resource.

### ISSUE THREAD

**Issue by glidenote** [hashicorp/terraform#3628]
when I use v0.6.6, ssh fingerprint does not work on DigitalOcean droplet resource.
v0.6.4 is no problem.

environment variable

```sh
export DIGITALOCEAN_TOKEN=xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
export TF_VAR_ssh_fingerprint=ab:ad:29:af:1d:4b:76:65:ea:25:9a:d7:fb:9a:26:e0
```

tf file

```
variable "ssh_fingerprint" {}

resource "digitalocean_droplet" "webapp" {
    image    = "centos-7-0-x64"
    name     = "webapp"
    region   = "sgp1"
    size     = "512mb"
    ssh_keys = [ "${var.ssh_fingerprint}" ]
}
```

error log.

```
terraform apply
digitalocean_droplet.webapp: Creating...
  image:                "" => "centos-7-0-x64"
  ipv4_address:         "" => "<computed>"
  ipv4_address_private: "" => "<computed>"
  ipv6_address:         "" => "<computed>"
  ipv6_address_private: "" => "<computed>"
  locked:               "" => "<computed>"
  name:                 "" => "webapp"
  region:               "" => "sgp1"
  size:                 "" => "512mb"
  ssh_keys.#:           "" => "1"
  ssh_keys.0:           "" => "ab:ad:29:af:1d:4b:76:65:ea:25:9a:d7:fb:9a:26:e0"
  status:               "" => "<computed>"
Error applying plan:

1 error(s) occurred:

* digitalocean_droplet.webapp: strconv.ParseInt: parsing "ab:ad:29:af:1d:4b:76:65:ea:25:9a:d7:fb:9a:26:e0": invalid syntax

Terraform does not automatically rollback in the face of errors.
Instead, your Terraform state file has been partially updated with
any resources that successfully completed. Please address the error
above and apply again to incrementally change your infrastructure.
```

DigitalOcean droplet resource doc https://terraform.io/docs/providers/do/r/droplet.html

> ssh_keys - (Optional) A list of SSH IDs or fingerprints to enable in the format [12345, 123456].

**Comment by apparentlymart** [comment]
Since v0.6.4 Terraform is using a new client library for DigitalOcean, from #3333.

@aybabtme, could this be a regression related to that client library switch?

**Comment by aybabtme** [comment]
I'll have a look

**Comment by aybabtme** [comment]
@apparentlymart I have a fix in #3633

**Comment by glidenote** [comment]
:+1: 
Thank you for quick response :smile:

**Comment by stack72** [comment]
@glidenote @apparentlymart can this issue be closed off now that #3633 was merged?

**Comment by glidenote** [comment]
:100:
